apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -702,6 +779,28 @@ def grade(element_html, data):
             -1 if num_initial_correct == len(submission) else num_initial_correct
         )
 
+        if feedback_type.startswith("first-wrong"):
+            for i, block in enumerate(student_answer):
+                if i >= num_initial_correct:
+                    break
+                block["badge_type"] = "badge-success"
+                block["icon"] = "fa-check"
+
+            if first_wrong != -1:
+                student_answer[first_wrong]["badge_type"] = "badge-danger"
+                student_answer[first_wrong]["icon"] = "fa-xmark"
+
+                for block in student_answer[first_wrong + 1 :]:
+                    block["badge_type"] = ""
+                    block["icon"] = ""
+
+        num_initial_correct, true_answer_length = grade_dag(
+            submission, depends_graph, group_belonging
+        )
+        first_wrong = (
```

Comment:
Nit: Use `None` as the sentinel value instead of -1.

Reply:
done